chaco/examples/demo/hyetograph.py

```diff
@@ -144,6 +144,8 @@ def start(self):
     )
 
 
+demo = Hyetograph()
+
+
 if __name__ == "__main__":
-    hyetograph = Hyetograph()
-    hyetograph.start()
+    demo.start()
```

Comment:
This is not working correctly (the demo works different when run in etsdemo vs standalone).  I imagine it is because etsdemo looks for `demo` and expects to call configure_traits or something, whereas here we really need to call start / we need to call `_perform_calculations` first...

Reply:
I updated the example to avoid this problem.  Note this example is referenced in the documentation, so perhaps the docs themselves should be changed to match?